test/evergreen.yml

```diff
@@ -1745,6 +1745,18 @@ tasks:
   # All cppsuite pull request tasks must supply the relative path to the config file as we are in
   # the cmake build working directory and the LD_LIBRARY_PATH is .libs.
 
+  - name: cppsuite-background-compact-default
+    tags: ["pull_request"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "cppsuite test"
+        vars:
+          test_config: debug_mode=(background_compact)
```

Comment:
Why does this one have the debug mode but the other doesn't?

Reply:
Given it's a default config can you please add the cursor copy debug mode too.

Reply:
The debug mode shortens the period at which we skip compacting certain files to every 5 seconds which allows us to run shorter tests while giving background compact more work to do. The long-running config is there to test the functionality without debug mode enabled which skips files for up to an hour.

Reply:
@luke-pearson What's the purpose of the cursor copy debug mode?

Reply:
Helps catch misuse of cursors, but in doing so slows things down, so we only use it on default tests.

Reply:
Okay, thanks!